apple/RNCWebView.m

```diff
@@ -311,13 +311,6 @@ - (WKWebViewConfiguration *)setUpWkWebViewConfig
     prefs.javaScriptEnabled = NO;
     _prefsUsed = YES;
   }
-  if (_allowUniversalAccessFromFileURLs) {
-    [wkWebViewConfig setValue:@TRUE forKey:@"allowUniversalAccessFromFileURLs"];
-  }
-  if (_allowFileAccessFromFileURLs) {
-    [prefs setValue:@TRUE forKey:@"allowFileAccessFromFileURLs"];
-    _prefsUsed = YES;
-  }
```

Comment:
I suppose the default values are false, but perhaps it's worth removing the if-conditions and setValue:@FALSE ?

Reply:
yea i am wondering if we just remoe this altogether so that there is no possible way to event set to true.

Reply:
I think you can just remove the if-condition and set them to false.

Reply:
done but as you mentioned -- default is false as it had to be added https://github.com/react-native-webview/react-native-webview/pull/1730